cmd/earthly/flags.go

```diff
@@ -323,6 +323,13 @@ func (app *earthlyApp) buildFlags() []cli.Flag {
 			Required:    false,
 			Destination: &app.orgName,
 		},
+		&cli.StringFlag{
+			Name:        "project",
+			EnvVars:     []string{"EARTHLY_PROJECT"},
+			Usage:       wrap("The name of the organization the satellite belongs to. ", "Required when using --satellite and user is a member of multiple organizations."),
```

Comment:
Update usage text? Seems that it was copy-pasted from `--org`.

Reply:
Updated.